src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message 'Bulldozer started its work' is not very informative. It would be better to indicate that the Bulldozer is the type of Machine that is working. Consider updating the message to something like 'Bulldozer is now working', to clearly indicate the type of Machine.